{
    "docs": "https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/compile.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test compile --python 2.7` [[explain](https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/compile.html)] failed with 29 errors:", 
            "output": "plugins/module_utils/commons.py:10:25: SyntaxError: def filter_none(instance: Any) -> Dict[str, Any]:\nplugins/module_utils/haproxy/client.py:37:32: SyntaxError: def __init__(self, base_url: str, api_version: str, username: str, password: str):\nplugins/module_utils/haproxy/client_acls.py:45:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_backend_switching_rules.py:45:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_backends.py:42:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_binds.py:45:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_configurations.py:28:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_frontends.py:42:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_http_request_rules.py:45:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_servers.py:45:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_ssl_certificates.py:39:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/client_transactions.py:41:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/haproxy/commons.py:10:25: SyntaxError: def filter_none(instance: Any) -> Dict[str, Any]:\nplugins/module_utils/haproxy/enums.py:12:26: SyntaxError: def create(cls, value: str):\nplugins/module_utils/haproxy/models.py:37:14: SyntaxError: algorithm: LoadBalancingAlgorithm\nplugins/module_utils/sonarqube/client.py:22:32: SyntaxError: def __init__(self, base_url: str, username: str, password: str):\nplugins/module_utils/sonarqube/client_settings.py:39:32: SyntaxError: def __init__(self, base_url: str, api_version: str, auth):\nplugins/module_utils/sonarqube/models.py:21:8: SyntaxError: key: str\nplugins/modules/haproxy/acl.py:144:19: SyntaxError: def get_acl(client: AclClient, index: int, parent_name: str, parent_type: str):\nplugins/modules/haproxy/backend.py:341:23: SyntaxError: def get_backend(client: BackendClient, name: str):\nplugins/modules/haproxy/backend_switching_rule.py:135:20: SyntaxError: def get_rule(client: BackendSwitchingRuleClient, index: int, frontend_name: str):\nplugins/modules/haproxy/bind.py:373:20: SyntaxError: def get_bind(client: BindClient, name: str, parent_name: str, parent_type: str):\nplugins/modules/haproxy/clean_transactions.py:66:31: SyntaxError: def cancel_transactions(module: AnsibleModule, client: TransactionClient):\nplugins/modules/haproxy/frontend.py:185:24: SyntaxError: def get_frontend(client: FrontendClient, name: str):\nplugins/modules/haproxy/http_request_rule.py:367:20: SyntaxError: def get_rule(client: HttpRequestRuleClient, index: int, parent_name: str, parent_type: str):\nplugins/modules/haproxy/server.py:371:22: SyntaxError: def get_server(client: ServerClient, name: str, parent_name: str, parent_type: str):\nplugins/modules/haproxy/ssl_certificate.py:111:27: SyntaxError: def get_certificate(client: SslCertificateClient, name: str):\nplugins/modules/haproxy/transaction.py:98:30: SyntaxError: def commit_transaction(module: AnsibleModule, client: TransactionClient, transaction_id: str, force_reload: bool):\nplugins/modules/sonarqube/settings.py:96:23: SyntaxError: def get_setting(client: SettingsClient, key: str, component: str):"
        }
    ], 
    "verified": false
}
